readthedocs/proxito/views/serve.py

```diff
@@ -406,6 +406,18 @@ def _register_broken_link(self, project, version, path, full_path):
             if not project.has_feature(Feature.RECORD_404_PAGE_VIEWS):
                 return
 
+            # This header is set from Cloudflare,
+            # it goes from 0 to 100, 0 being low risk,
+            # and values above 10 are bots/spammers.
+            # https://developers.cloudflare.com/ruleset-engine/rules-language/fields/#dynamic-fields.
+            threat_score = int(self.request.headers.get("X-Cloudflare-Threat-Score", 0))
```

Comment:
We may need to handle the exception here in case the string is not able to be converted to int.

Reply:
Cool! 👍🏼